### PR TITLE
Doc: Restore links to importing MIBs info

### PR DESCRIPTION
**Pull request by karenzone**
Related: #3

### CODE REVIEW

File: docs/input-snmp.asciidoc [hunk 1]
```diff
@@ -74,10 +72,8 @@ Metadata fields follow a specific naming convention when <<plugins-{type}s-{plug
 
 This plugin already includes the IETF MIBs (management information bases), and you do not need to import them.
 If you need additional MIBs, you need to import them.  
-Check out "Importing MIBs" for instructions.
+Check out link:{logstash-ref}/plugins-input-snmp.html[Importing MIBs] for info.
```
Comment (karenzone) [inline docs/input-snmp.asciidoc:75]:
```suggestion
Check out link:{logstash-ref}/plugins-integrations-snmp.html[Importing MIBs] for info.
```

File: docs/input-snmptrap.asciidoc [hunk 2]
```diff
@@ -90,10 +90,7 @@ which variable binding in the list caused the error
 
 This plugin already includes the IETF MIBs (management information bases), and you do not need to import them.
 If you need additional MIBs, you need to import them.  
-Check out "Importing MIBs" for instructions.
-
-// link:{logstash-ref}/plugins-input-snmp.html[Importing MIBs]
-// ToDo: Add link to MIBs content after generated file exists.
+Check out link:{logstash-ref}/plugins-input-snmp.html[Importing MIBs] for info.
```
Comment (karenzone) [inline docs/input-snmptrap.asciidoc:93]:
```suggestion
Check out link:{logstash-ref}/plugins-integrations-snmp.html[Importing MIBs] for info.
```

**Comment by karenzone** [comment]
@logstashmachine bump patch

**Review by edmocosta — APPROVED** [review]
LGTM 🎉

**Comment by karenzone** [comment]
9614

**Pull request merged**